shared/middleware/helpers/redis_table.rb

```diff
@@ -47,6 +48,8 @@ def initialize(redis, pub_sub_api, shard_id, table_name, expire_in = nil)
   # @return [Hash] The inserted value, including the new :id field.
   def insert(value, ignored_ip=nil)
     new_id = next_id
+    new_uuid = SecureRandom.uuid
+    value = value.merge({'uuid' => new_uuid})
```

Comment:
Do you mean that we have to create a UUID here and also preserve it across an update?  Or that we merge the UUID at the top of the insert method, and the ID at the end of it?

If the latter, I suppose we could merge the ID in before setting the value in Redis (so we duplicate the ID in the key and value).